src/main/java/org/sopt/seminar1/Diary.java

```diff
@@ -6,9 +6,21 @@ public class Diary {
 
     private final String body;
 
+    private boolean isDelete;
+    private  int updateCount;
```

Comment:
```suggestion
    private boolean isDelete = false;
    private  int updateCount=0;
```
이렇게 지정해도 동작했을 것 같긴한데, 생성자 두 개 만들어서 구현했을 때 더 좋은 점들이 있나요!?